dev-tools/cmd/module_include_list/module_include_list.go

```diff
@@ -0,0 +1,157 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/elastic/beats/dev-tools/mage"
+	"github.com/elastic/beats/licenses"
+)
+
+var usageText = `
+Usage: module_include_list [flags] [module-dir]
+  module_include_list generates a list.go file containing import statements for
+  the module and its metricset packages. The file is usually written to the 
+  include/list.go in the Beat's root directory.
+Options:
+`[1:]
+
+var (
+	license string
+	pkg     string
+	outFile string
+)
+
+func init() {
+	flag.StringVar(&license, "license", "ASL2", "License header for generated file.")
+	flag.StringVar(&pkg, "pkg", "include", "Package name.")
+	flag.StringVar(&outFile, "out", "include/list.go", "Output file.")
+	flag.Usage = usageFlag
+}
+
+func main() {
+	log.SetFlags(0)
+	flag.Parse()
+
+	license, err := licenses.Find(license)
+	if err != nil {
+		log.Fatalf("Invalid license specifier: %v", err)
+	}
+
+	args := flag.Args()
+	if len(args) != 1 {
+		log.Fatal("module-dir must be passed as an argument.")
+	}
+	dir := args[0]
+
+	// Find modules and metricsets.
```

Comment:
Perhaps you can call this dataset instead of metricset. I assume this is for FB and MB.

Reply:
Will change to dataset.